src/foundation/Icon/Generated/IconStrikethrough.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { memo } from "react";
+import { GeneratedIconProps } from "@foundation/Icon/IconProps";
+import { IconSize, IconSizeMap } from "@foundation/Icon/IconSize";
+
+function IconStrikethrough(props: GeneratedIconProps): React.ReactElement<GeneratedIconProps> {
+    const customClassName = [
+        "tw-flex tw-items-center tw-justify-center tw-fill-current",
+        IconSizeMap[props.size || IconSize.Size16],
+    ].join(" ");
+    return (
+        <svg
+            viewBox="0 0 17 17"
+            fill="none"
```

Comment:
same here with `currentColor` and remove the fill props below

Reply:
fixed